native/src/components/NearbyCities.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { ReactElement } from 'react'
+import { TFunction } from 'react-i18next'
+import { Button } from 'react-native-elements'
+import Icon from 'react-native-vector-icons/MaterialIcons'
+import styled, { useTheme } from 'styled-components/native'
+
+import { CityModel } from 'api-client'
+
+import useUserLocation from '../hooks/useUserLocation'
+import getNearbyCities from '../utils/getNearbyCities'
+import CityEntry from './CityEntry'
+
+const NearbyMessageContainer = styled.View`
+  padding: 7px;
+  flex-direction: row;
+  justify-content: space-between;
+`
+const RetryButtonContainer = styled.View`
+  flex-direction: column;
+  height: 46px;
```

Comment:
it's recommended to use a padding for the container that surrounds the icon instead of a fixed height, but this is also optional

Reply:
Feel free to tell me what to change, didn't do this though (at least not in this PR :stuck_out_tongue: )

Reply:
add a padding and remove height ;) as in the `SearchInput.tsx` but its fine in this pr. just a general hint

Reply:
I think this won't work here since the children is rendered conditionally only if the user location is currently not loading. If we just set padding, the height will therefore be different if the button is not rendered  than when it is rendered. Therefore the UI will jump. Will keep this as it is.